app/controllers/provider/passwords_controller.rb

```diff
@@ -28,15 +28,15 @@ def destroy
   end
 
   def show
-    if params[:password_reset_token].present? && session[:password_reset_token].blank?
+    if params.permit!.to_h[:password_reset_token].present? && session[:password_reset_token].blank?
```

Comment:
You probably want to have

```ruby
def password_params
  @password_params ||= params.permit(:password_reset_token, {user: { :password, :password_confirmation})
end
```

Then reuse it in `#update` and `#show` actions

Reply:
@hallelujah Changes done